target/linux/generic/config-5.4

```diff
@@ -4120,7 +4120,7 @@ CONFIG_PRINT_STACK_DEPTH=64
 CONFIG_PROC_FS=y
 # CONFIG_PROC_KCORE is not set
 # CONFIG_PROC_PAGE_MONITOR is not set
-CONFIG_PROC_STRIPPED=y
+CONFIG_PROC_STRIPPED=n
```

Comment:
Wrong syntax (just check the rest of the file ...)

Reply:
whuuups. sry. changed.